packages/rx_bloc/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [5.1.0]
```

Comment:
When upgrading dev dependencies we do not need to bump the version of the package